install-cockroachdb.md

```diff
@@ -182,7 +182,10 @@ $(document).ready(function(){
         <p>A <a href="http://golang.org/doc/code.html">Go environment</a> with a 64-bit version of Go 1.8.</p>
       </li>
       <li>
-        <p><a href="https://www.gnu.org/software/bash/">Bash</a></p>
+        <p><a href="https://www.gnu.org/software/bash/">Bash</a>, preferably version 4+, though the older Bash version that ships with macOS is known to work.</p>
```

Comment:
nit: Let's use the same formulation for mac and linux. I prefer the linux formulation, but using the plural `versions` like we do for `CMake`:

> Versions 4+ are preferred, but later releases from the 3.x series are also known to work.</p>